src/SecurityManager.ts

```diff
@@ -352,14 +366,19 @@ export async function accessSecretStorage(func = async () => { }, forceReset = f
                 }
                 console.log("Setting dehydration key");
                 await cli.setDehydrationKey(secretStorageKeys[keyId], dehydrationKeyInfo, "Backup device");
-            } else {
+            } else if (!keyId) {
                 console.log("Not setting dehydration key: no SSSS key found");
+            } else {
+                console.log("Not setting dehydration key: feature disabled");
```

Comment:
I'm slightly curious if these two should be warnings instead of info

Reply:
Ah, I'll upgrade the key not found path to warning at least.